scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -766,6 +766,8 @@ object Pat {
     })
   }
   @ast
+  class Assign(name: Term.Name, rhs: Pat) extends Pat
```

Comment:
could you also extend `Tree.WithBody`, just like `Term.Assign`? (I wonder why Term.Assign allows not just Name).

Reply:
Done!